tests/rptest/tests/pandaproxy_test.py

```diff
@@ -2129,3 +2132,130 @@ def test_upgrade_and_enable_basic_auth(self, base_release: tuple[int, int],
         unique_versions = wait_for_num_versions(self.redpanda, 1)
         assert old_version_str not in unique_versions
         self.check_usage()
+
+
+class VerifiablePandaproxyProducer(threading.Thread):
```

Comment:
I recall in a sync between us and @BenPope , we decided that the initial versions of a VerifiablePandaproxy Producer and Consumer will simply issue REST requests repeatedly on the orchestrator node since we already do that.